### ISSUE TITLE
Client Crash on getIcon()

### ISSUE THREAD

**Issue by Robotia** [covers1624/PowerConverters#30]
A player on my server experienced this crash related to the getIcon clientside function for rendering a power converter block. There is an NPE, I think it's because the TileEntity at the specific location(line 37)  is null.

Crash Report: http://pastebin.com/Zbftj620

```java
        @Override
        @SideOnly(Side.CLIENT)
        public IIcon getIcon(IBlockAccess world, int x, int y, int z, int side) {
                int offset = ((TileEntityBridgeComponent<?>) world.getTileEntity(x, y, z)).isSideConnectedClient(side) ? 1 : 0; // line 37
                return icons[world.getBlockMetadata(x, y, z) * 2 + offset];
        }

```

Any ideas @covers1624 ?

**Comment by covers1624** [comment]
Probably due to that, im going to do a rewrite soon, have just officially taken over cb's mods so once they are done 1.9 train for powerconverters.

**Comment by covers1624** [comment]
There will be a build soon, added checks to avoid this.

**Comment by Rubbertjuh** [comment]
Will this build be available for 1.7.10 as well ? or 1.9 only?

**Comment by covers1624** [comment]
1.7 when i finish things off and do a release
